méli-mélo/th-empathy/empathy.html

```diff
@@ -0,0 +1,93 @@
+---
+title: Empathy theme
+dateModified: 2024-04-30
+description: Empathy CSS styles
+lang: en
+pageclass: cnt-wdth-lmtd
+css:
+- style.css
+- https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/méli-mélo/2025-04-nahanni.css
+script:
+- https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/méli-mélo/2025-04-nahanni.js
+layout: without-h1
+---
+<h1 property="name" id="wb-cont" dir="ltr" class="empathy">{{ page.title }}</h1>
```

Comment:
Content optimization suggestion, there is no need to set the text direction here from left to right because it is already like that by default.

```suggestion
<h1 property="name" id="wb-cont" class="empathy">{{ page.title }}</h1>
```